doc/lava_usecase.rst

```diff
@@ -0,0 +1,95 @@
+===============================
+Use case: setup SQUAD with LAVA
+===============================
+
+Introduction
+------------
+
+Once SQUAD installation is complete, a typical use case would be integrating
+it with a LAVA instance. The purpose of this use case is to describe a
+step-by-step set up of SQUAD with LAVA to submit and fetch jobs. If you haven't
+yet set up SQUAD, take a step back and follow :ref:`production_install_ref_label`
+
+
+Setting up a LAVA instance
+--------------------------
+
+LAVA has its own extensive documentation on how to get a server up and running.
+If you don't have it already, please refer to `LAVA installation`_ to configure
+yourself one. From this point it's taken that you have enough access to a
+running LAVA v2 instance that jobs can be submitted to and fetched from.
+
+.. _`LAVA installation`: https://validation.linaro.org/static/docs/v2/installing_on_debian.html#debian-installation
+.. _`LAVA authentication tokens`: https://validation.linaro.org/static/docs/v2/first_steps.html?highlight=token#authentication-tokens
+
+Creating a Backend for a LAVA instance
+--------------------------------------
+
+Log in into SQUAD admin view (``/admin``) and access ``ci > backends > add backend``
+for inclusion form. Fill up accordingly:
+
+- name: name of the backend. Example: validation.linaro.org
+- url: LAVA RPC2 endpoint, it's how SQUAD will communicate with LAVA. Example: https://validation.linaro.org/RPC2
+- username: a LAVA user with enough access to submit jobs
+- token: a generated token for the user above, used to securely connect to the LAVA instance. See `LAVA authentication tokens`_ for details
+- implementation type: leave it as ``lava``
+- backend settings: used to spare specific settings for LAVA instances. For details see :ref:`backend_settings_ref_label` 
+- poll interval: number of minutes to wait before fetching a job from LAVA
+- max fetch attempts: max number of times SQUAD will attempt to fetch a job from LAVA
+- poll enabled: if this is disabled SQUAD will not try to poll jobs from LAVA 
+
+
+Creating a Project in SQUAD
+---------------------------
+
+SQUAD needs minimal data to start working with LAVA: Group and Project.
+By logging in the admin view, go to ``core > groups > add`` to add a new
+group and ``core > projects > add`` to add a new project. These are trivial
+to create, but please feel free to contact us if any help is needed.
+
+
+Start SQUAD workers and listener
+--------------------------------
+
+Start up the workers to allow SQUAD processes to comunicate with each other::
+
+    celery -A squad worker -Q celery,reporting_queue --max-tasks-per-child=5000 --max-memory-per-child=1500000 --loglevel INFO
```

Comment:
I agree. The table was hard to see on readthedocs because it missed a '+' on it. Now it's straightforward to get that from the doc. Removing this and the `squad-admin listen` part